addons/website/static/src/builder/plugins/translation_plugin.js

```diff
@@ -31,18 +29,13 @@ export function getTranslationEditableEls(rootEl) {
  */
 function findOEditable(containerEl) {
     const isOEditable = (node) => {
-        while (node) {
-            if (node.className && typeof node.className === "string") {
-                if (node.className.includes("o_not_editable")) {
-                    return false;
-                }
-                if (node.className.includes("o_editable")) {
-                    return true;
-                }
-            }
-            node = node.parentNode;
+        const closestEditableOrNotEditable = node.closest(
```

Comment:
nope: you want to check if the element is a content editable; el.isContentEditable or if el.matches(".o_editable_media") +check? or if  el.matches(".o_editable_attribute") cad, si l'element est une zone editbale